packages/eas-cli/src/submit/ios/AppSpecificPasswordSource.ts

```diff
@@ -11,15 +17,62 @@ interface AppSpecificPasswordUserDefinedSource extends AppSpecificPasswordSource
   appSpecificPassword: string;
 }
 
+export type AppSpecificPassword = {
```

Comment:
```suggestion
export interface AppSpecificPasswordCredentials {
```
- I think we prefer interfaces over types
- IMO the `Credentials` suffix will make this name more accurate